internal/types/document.go

```diff
@@ -93,6 +93,10 @@ func NewDocument(pairs ...any) (*Document, error) {
 		}
 	}
 
+	if err := doc.ValidateCmd(); err != nil {
+		return nil, fmt.Errorf("types.NewDocument: %w", err)
+	}
+
```

Comment:
Not sure if my reasoning behind command documents is correct

Reply:
This is a good question! Not every document is a command document. I think `ValidateCmd` should be called from handlers. @AlekSi, please correct me if I'm wrong.